AppBidding-MoPub/AppBidding-MoPub/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+
+
+#import <UIKit/UIKit.h>
+#import "MoPub.h"
```

Comment:
I believe `MoPub.h` is not used in this context.

Reply:
Without MoPub.h in the AppDelegate.j we can't create the sdkConfig variable that is in AppDelegate.m

Reply:
MoPub.h moved to AppDelegate.m